staging/src/k8s.io/apiextensions-apiserver/pkg/controller/openapi/builder.go

```diff
@@ -39,7 +39,7 @@ import (
 	"k8s.io/kube-openapi/pkg/util"
 
 	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions"
-	generatedopenapi "k8s.io/apiextensions-apiserver/pkg/generated/openapi"
```

Comment:
is the change of generated openapi package path intended? consider backward-compatibility?

Reply:
Yes, it is. We had two specs, used the wrong, old one.

This code here is considered internal. Nothing depends on it.